provisioning/roles/qa_framework/vars/Unix.yaml

```diff
@@ -0,0 +1 @@
+python_executable: /usr/local/bin/python3.10
```

Comment:
All Unix systems always have python3.10?

Reply:
Every Linux System supported by the framework Amazon Linux 2, CentOS8, Ubuntu22, and RedHat8 have python3.10. However, this does not include other Unix-based systems like macOS. Those include their own variables.
We should consider change this name to avoid confusion

Reply:
Replace it by `linux` in c86a9d1